content/1.7/back-office/dashboard/dashboard-view.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Dashboard view
+weight: 1
+---
+
+# Dashboard view
+
+Dashboard is the first default page on the back office, in this page the shop performance is displayed.
+
+## Call to action
+
+ - **Demo mode**: this is a toggle switch, it can turn on or off demo mode stats. When in demo mode shows fake stats, when demo mode is turned off, shows the real stats in the dashboard. Has a tooltip: "This mode displays sample data so you can try your dashboard without real numbers."
+
+Below there is a timeline range which can be modified to show the dashboard according to the needed timeline. It can be turned on for:
+
+ - **Day**
+ - **Month**
+ - **Year** - by default it is set to a year.
+ - **Day-1**
+ - **Month-1**
+ - **Year-1**
+
+On the right a calendar date selection is displayed (the date depends on which time frame is selected for the dashboard), when pressed on it, the calendar extends.
+
+After the calendar is extended, new row appears with 4 fields.
+
+In the first 2 fields dates are present, you can edit the date, select the year, month and days from which to which you want the data to be loaded.
+
+On the right there is a field "Date range", has 2 fields "from" and "to" for date inputs.
+
+There is also a checkbox "Compare to", when pressed, a new row appears with the same 2 fields for dates, on the right there's a drop field, which allows to select the following:
+ - **Previous period**
+ - **Previous year**
+ - **Custom**
+
+If the selected dates are incorrect, will show the following error: "The selected date range is not valid."
+
+## Call to action
+
+2 buttons are located at the bottom:
+ 
+ - **Cancel** - when pressed, closes the tab, displayed the result.
+ - **Apply** - when pressed, closes the tab, displayed the result.
+
+## Fields
+
+By default there are 8 fields in the dashboard:
```

Comment:
```suggestion
There is 2 columns in the dashboard: 
- the left column: this column presents the data from the Dashboard Activity module
- the central column: there are three blocks of content in this column, which are handled by modules
```

Reply:
Sorry, I forgot to tell you that the right column with the blocks "prestashop news", "prestashop update", and "we stay by your side" will disappear in the V8